src/components/Installers.astro

```diff
@@ -54,13 +54,17 @@ import { installers } from "../data/installers.ts";
         }
     }
 
+	details {
+		display: flex;
+	}
+
     summary {
-        margin: 2em 0 0 0;
-        width: 100%;
+        margin: 2em auto 0 auto;
+		width: min-content;
```

Comment:
Mixed whitespace I think

![image](https://github.com/user-attachments/assets/07e2b71d-f961-466d-9972-ec1f8aab8d63)

Reply:
Sorry my editor is allergic to bad whitespace decisions and like to spread tab superiority /s

Fixed now :)